opensearch-operator/controllers/suite_test_helpers.go

```diff
@@ -175,8 +176,17 @@ func ComposeOpensearchCrd(clusterName string, namespace string) opsterv1.OpenSea
 			},
 			NodePools: []opsterv1.NodePool{{
 				Component: "master",
-				Replicas:  3,
-				DiskSize:  "32Gi",
+				Pdb: opsterv1.PdbConfig{
+					EnablePDB: true,
+					MinAvailable: &intstr.IntOrString{
+						IntVal: 3,
+					},
+					MaxUnavailable: &intstr.IntOrString{
+						IntVal: 3,
+					},
```

Comment:
This does not make any sense. In the code you state and check that only one of them can be set, but here you set both.